src/openzaak/components/besluiten/admin.py

```diff
@@ -35,10 +56,36 @@ class BesluitInformatieObjectInline(EditInlineAdminMixin, admin.TabularInline):
     fk_name = "besluit"
 
 
+class BesluitForm(forms.ModelForm):
+    class Meta:
+        model = Besluit
+        fields = "__all__"
+
+    def clean(self):
+        cleaned_data = super().clean()
+
+        if not cleaned_data.get("_besluittype_url") and not cleaned_data.get(
+            "_besluittype"
+        ):
+            raise forms.ValidationError(
+                "Je moet een besluittype opgeven: "
+                "selecteer een besluittype uit de catalogus of vul een externe URL in."
+            )
+
+        if not cleaned_data.get("_zaak_url") and not cleaned_data.get("_zaak"):
```

Comment:
`zaak` is not a required field in the `Besluit` API, so it shouldn't be required also in the Admin form. Can you also check please if all other checks are added for required fields?

Reply:
Ah you're right.  I didn't realize `zaak` had `blank=True, null=True` in the `FkOrUrlField`.  I'll remove this if statement since it's not actually required.